Manifest.toml

```diff
@@ -0,0 +1,1162 @@
+# This file is machine-generated - editing it directly is not advised
+
+[[AbstractAlgebra]]
```

Comment:
We shouldn't include a Manifest

Reply:
Reason I added the Manifest this time was cause I want to use SBML#master for CI, which is several commits ahead of https://github.com/LCSB-BioCore/SBML.jl/tree/v0.10.1
Got me really confused yesterday.

Reply:
I forgot the customs for version updating. How to deal with this? Once Mirek bumps the SBML version, we

1. bump the SBML compat
2. bump the SBMLtk patch version
3. remove Manifest
4. merge with main?